examples/drivenet/drivenet_open_loop.ipynb

```diff
@@ -0,0 +1,354 @@
+    "from l5kit.visualization import TARGET_POINTS_COLOR, PREDICTED_POINTS_COLOR, draw_trajectory\n",
+    "from l5kit.drivenet.model import DriveNetModel\n",
+    "from l5kit.kinematic import AckermanPerturbation\n",
+    "from l5kit.random import GaussianRandomGenerator\n",
+    "\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare Data path and load cfg\n",
+    "\n",
+    "By setting the `L5KIT_DATA_FOLDER` variable, we can point the script to the folder where the data lies.\n",
+    "\n",
+    "Then, we load our config file with relative paths and other configurations (rasteriser, training params...)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set env variable for data\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
+    "dm = LocalDataManager(None)\n",
+    "# get config\n",
+    "cfg = load_config_data(\"./drivenet_config.yaml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load The Model\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_path = \"/tmp/drivenet.pt\"\n",
+    "device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
+    "model = torch.load(model_path).to(device)\n",
+    "model = model.eval()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the Evaluation Data\n",
+    "This is almost the exact same code you've already seen in the [training notebook](./drivenet_train.ipynb). Apart from the different dataset we load, the biggest difference is that **we don't perturb our data here**.\n",
+    "\n",
+    "When performing evaluation we're interested in knowing the performance on the annotated data, not on perturbed ones."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ===== INIT DATASET\n",
+    "eval_cfg = cfg[\"val_data_loader\"]\n",
+    "rasterizer = build_rasterizer(cfg, dm)\n",
+    "eval_zarr = ChunkedDataset(dm.require(eval_cfg[\"key\"])).open()\n",
+    "eval_dataset = EgoDataset(cfg, eval_zarr, rasterizer)\n",
+    "eval_dataloader = DataLoader(eval_dataset, shuffle=eval_cfg[\"shuffle\"], batch_size=eval_cfg[\"batch_size\"], \n",
+    "                             num_workers=eval_cfg[\"num_workers\"])\n",
+    "print(eval_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluation loop\n",
+    "\n",
+    "Here, we loop through the data and store predicted and annotated trajectories (positions + yaws)\n",
+    "\n",
+    "Note: we're not taking into account availability here. We acknowledge this can reflects in lower metric."
```

Comment:
What does this mean?
Also: reflect

Reply:
I'm not storing availabilities and use them during metric computation to filter out some elements. It's not a big deal, I'll just remove the sentence and we can address it later if we want to